src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -243,6 +262,49 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
       />
     );
 
+  const plannedGraduationYear =
+    isFacStaff || (isAlumni && !profile.PlannedGradYear?.length) ? null : (
+      <ProfileInfoListItem
+        title={'Planned Graduation Year:'}
+        contentText={
+          myProf && isStudent && !isAlumni ? (
+            <Grid container spacing={0} alignItems="center">
+              <Grid item>
+                {!profile.PlannedGradYear?.length ? 'Deciding' : profile.PlannedGradYear}
+              </Grid>
+              <Grid item>
+                <UpdatePlannedGraduationYear />
+              </Grid>
+            </Grid>
+          ) : profile.IsPlannedGradYearPrivate ? (
+            PRIVATE_INFO
+          ) : !profile.PlannedGradYear?.length ? (
+            'Deciding'
+          ) : (
+            profile.PlannedGradYear
+          )
+        }
+        // Always public for now
+        // ContentIcon={
+        //   myProf && (
+        //     <FormControlLabel
+        //       control={
+        //         <Switch
+        //           onChange={handleChangePlannedGraduationYearPrivate}
+        //           checked={!isPlannedGraduationYearPrivate}
+        //         />
+        //       }
+        //       label={isPlannedGraduationYearPrivate ? 'Private' : 'Public'}
+        //       labelPlacement="bottom"
+        //       disabled={!isOnline}
+        //     />
+        //   )
+        // }
```

Comment:
```suggestion
        // Always public for now
        // ContentIcon={
        //   myProf && (
        //     <FormControlLabel
        //       control={
        //         <Switch
        //           onChange={handleChangePlannedGraduationYearPrivate}
        //           checked={!isPlannedGraduationYearPrivate}
        //         />
        //       }
        //       label={isPlannedGraduationYearPrivate ? 'Private' : 'Public'}
        //       labelPlacement="bottom"
        //       disabled={!isOnline}
        //     />
        //   )
        // }
```

remove if not used

Reply:
In this case that we will soon implement the change privacy for the planned graduation year once the new privacy table is finished, should I leave the code for the toggle commented or remove and add it all again later?

Reply:
removed